### PR TITLE
Completed JS Exercises 

### PR DESCRIPTION
**Pull request by tonymbbrown**
Completed the assigned JS Exercises

### CODE REVIEW

**Comment by tonymbbrown** [comment]
js-exercise2-screenshot

**Comment by tonymbbrown** [comment]
w3_javascript_09_19_10_10
w3_javascript_09_19_10_10_number_2

Attached screenshots for w3schools Javascript exercises.

**Comment by tonymbbrown** [comment]
js-exercise4-screenshot
js-exercise5-screenshot
Screenshots for js-exercise4 and js-exercise5.

File: tony/js/js-exercise5.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            JS Exercise 5
+        </title>
+        <script>
+            function addToWishlist(productName)
+            {
+                var listNode = document.createElement("li");
+                listNode.appendChild(document.createTextNode(productName));
+
+                var wishlist = document.getElementById("wishlist");
+                wishlist.appendChild(listNode);
```
[the remark this answers is not in the document]
Comment (tonymbbrown) [reply tony/js/js-exercise5.html:14]: js-exercise5-screenshot
Just pushed this functionality

**Review by myolap — COMMENTED** [review]
You should move your code to the js file instead of putting it in the html.

**Review by madhanramu — APPROVED** [review]

**Pull request merged**